include/pfs/drp/stella/FiberTraces.h

```diff
@@ -57,128 +49,292 @@
 //#define __DEBUG_XCENTERS__
 #define DEBUGDIR "/Users/azuri/spectra/pfs/2014-11-02/debug/"// /home/azuri/entwicklung/idl/REDUCE/16_03_2013/"//stella/ses-pipeline/c/msimulateskysubtraction/data/"//spectra/elaina/eso_archive/red_564/red_r/"
 
-namespace afwGeom = lsst::afw::geom;
 namespace afwImage = lsst::afw::image;
 namespace pexExcept = lsst::pex::exceptions;
 
 using namespace std;
 
 namespace pfs { namespace drp { namespace stella {
 /**
- * \brief Describe a single fiber trace
+ * @brief Describe a single fiber trace
```

Comment:
This is not a header reordering change, and doesn't belong on this ticket.

You may merge these changes this time.